src/Tribe/Plugin_Download_Notice.php

```diff
@@ -0,0 +1,72 @@
+<?php defined( 'WPINC' ) or die;
+
+if ( ! class_exists( 'Tribe__Plugin_Download_Notice' ) ) {
+
+	/**
+	 * Shows an admin notice telling users which requisite plugins they need to download
+	 */
+	class Tribe__Plugin_Download_Notice {
```

Comment:
Same thing here, move these methods to the _Dependency_ class, and we also should be using the new `Tribe__Admin__Notice` class, with the helper `tribe_notice`.